java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/git/Git.java

```diff
@@ -61,6 +61,7 @@ protected void pushRepository(Path path, String httpTransportUrl, String commit,
             Log.infof("Updating current origin of %s to %s", jConfig.getString("remote", "origin", "url"),
                     httpTransportUrl);
             jConfig.setString("remote", "origin", "url", httpTransportUrl);
+            jConfig.setBoolean("http", null, "sslVerify", false);
```

Comment:
We definitely don't want to hard code this, it should be optional for testing/dev.

Reply:
I needed this to get the demo with gitlab working.